projects/jupyter-server-ydoc/jupyter_server_ydoc/handlers.py

```diff
@@ -291,6 +292,16 @@ async def on_message(self, message):
         """
         On message receive.
         """
+        if message == "save_to_disc":
```

Comment:
I think that we should use a custom message type (see [here](https://github.com/yjs/y-protocols/blob/master/PROTOCOL.md#combining-protocols)). Maybe `2` followed by `save`?